README.rst

```diff
@@ -1,65 +1,60 @@
+  and install OpenVPN
 
-* Download openvpn-gui-1.0.X.exe and save it in OpenVPN's bin folder.
-  Default is "C:\Program Files\OpenVPN\bin\". You must put it in this folder
+* Download OpenVPN GUI of your choice and save it in OpenVPN's bin folder.
+  Default is *C:\\Program Files\\OpenVPN\\bin\\*. You must put it in this folder
   because OpenVPN GUI depends on the OpenSSL DLLs installed in this folder by
   OpenVPN.
 
-* Create a xxxx.ovpn config-file with your favorite texteditor and save it in:
-  C:\Program files\OpenVPN\config\. You should NOT use the "log" or "log-append"
-  options as OpenVPN GUI redirect the normal output to a log file itself. 
-  There is a sample config files in the "sample-config" folder. Please
-  refer to the OpenVPN project homepage for more information regarding 
-  creating the configuration file. http://openvpn.net/
+Configuring OpenVPN GUI to start on Windows logon
+*************************************************
+
+Put a short-cut to openvpn-gui.exe in your *Start -> All Program -> StartUp*
+folder if you want the gui to start automatically when you logon to Windows.
 
-* Put a short-cut to openvpn-gui-1.0-X.exe in your 
-  "Start->All Program->StartUp" folder if you want the gui started automatically
-  when you logon to Windows.
+Adding an OpenVPN configuration file
+************************************
 
-* Start the GUI by double-clicking the openvpn-gui-1.0.X.exe file. You need to 
-  be Administrator the first time you run OpenVPN GUI for it to create its 
-  registry keys. After that you don't have to be administrator just to run the 
-  GUI, however OpenVPN requires the user to be administrator to run!
+To launch a VPN connections using OpenVPN GUI you must add an OpenVPN
+configuration file with .ovpn suffix and put it to *C:\\Program
+Files\\OpenVPN\config\\*. Any text editor (e.g. notepad.exe) can be used to
+create this file. You should NOT use the *log* or *log-append* options in the
+OpenVPN configuration file as OpenVPN GUI redirects the normal output to a log
+file itself. There are sample config files in the *sample-config* folder. Please
+refer to the `OpenVPN project homepage <https://openvpn.net>`_ for more information regarding creating
+the configuration file.
 
 Using OpenVPN GUI
 #################
 
-When OpenVPN GUI is started your config folder (C:\Program Files\OpenVPN\config)
-will be scanned for .ovpn files, and an icon will be displayed in the taskbar's
-status area.
+When OpenVPN GUI is started your OpenVPN config folder
+(*C:\\Program Files\\OpenVPN\\config*) will be scanned for .ovpn files and the
+OpenVPN GUI icon will appear in the system tray. Each OpenVPN configuration file
+shows up as a separate menu item in the OpenVPN GUI tray, allowing you to
+selectively connect to and disconnect to your VPNs.
 
-If you do not have any openvpn connection running, the config dir will be
+If you do not have any OpenVPN connection running, the config dir will be
```

Comment:
For v11 the rescan happens even if there are connections are running, so new configs are immediately recognized.